src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml

```diff
@@ -50,19 +51,15 @@
         {
             var item = categoryCount;
 
-            @Html.Hidden("categoryid_to_calculate" + categoryCount, Model.TotalCategoryValues.ElementAt(categoryCount).CategoryId)
-            @Html.HiddenFor(m => m.TotalCategoryValues[item].CategoryDisplay)
-            var receivedId = "Received" + Model.TotalCategoryValues.ElementAt(categoryCount).CategoryId + "Total";
-            var reusedId = "Reused" + Model.TotalCategoryValues.ElementAt(categoryCount).CategoryId + "Total";
+            //var receivedId = "Received" + Model.TotalCategoryValues.ElementAt(categoryCount).CategoryId + "Total";
+            //var reusedId = "Reused" + Model.TotalCategoryValues.ElementAt(categoryCount).CategoryId + "Total";
```

Comment:
if that's not going to be needed, would you mind deleting it?

Reply:
I think deleted now